monitoring/publishing/models.py

```diff
@@ -6,28 +6,60 @@
 
 class GridSite(models.Model):
     fetched = models.DateTimeField(auto_now=True)
-    name = models.CharField(max_length=255, primary_key=True)
+    SiteName = models.CharField(max_length=255, primary_key=True)
```

Comment:
Is it a django thing that these kinds of variables need to be CamelCase?

Reply:
The name of the field needs to match the relevant database, but this only really applies to the unmanaged models that are designed to be used to pull from the APEL Repository dbs. Probably not needed here (though if we revert we'd have to change any other references).

Reply:
Tracking in #19